synapse/storage/databases/main/schema/delta/58/07add_method_to_thumbnail_constraint.sql.postgres

```diff
@@ -20,14 +20,14 @@
  */
 
 -- add new index that includes method to local media
-INSERT INTO background_updates (update_name, progress_json) VALUES
-  ('local_media_repository_thumbnails_method_idx', '{}');
+INSERT INTO background_updates (ordering, update_name, progress_json) VALUES
+  (5805, 'local_media_repository_thumbnails_method_idx', '{}');
```

Comment:
As you said the numbers are arbitrary, but was there a particular reason this doesn't match the delta number (5807)?

Reply:
I... got confused?